common/source/docs/common-telemetry-landingpage.rst

```diff
@@ -9,7 +9,7 @@ transmitter) using telemetry. Follow the links below for configuration
 information based upon your set-up.
 
 .. note:: Some RC systems incorporate telemetry in addition to RC control
-
+.. note:: Ardupilot versions before 4.2 will automatically fall back to back to mavlink1 if mavlink 2 is selected and no mavlink 2 data is received from the groundstation
```

Comment:
```suggestion
.. note:: Ardupilot will automatically fall back to back to MAVLink1 if MAVLink 2 serial port protocol is selected and no MAVLink 2 data is received from the groundstation in response
```

Reply:
I put changing this wording in the future wiki changes list for 4.2....with this change it can go into main wiki

Reply:
looks good